Controllers/Mobile/v1/ProjectsController.cs

```diff
@@ -185,8 +185,8 @@ public async Task<IActionResult> SubmitBidAsync(long projectId, [FromBody] BidIn
             Project? storedProject = await projectService.FindProjectWithBidsAsync(projectId);
             if (storedProject == null)
                 return NotFound(CreateErrorResponse(StatusCodes.Status404NotFound.ToString(), "Project not found."));
-            if (storedProject.Status != Constants.PROJECT_STATUS_AVAILABLE)
-                return Conflict(CreateErrorResponse(StatusCodes.Status409Conflict.ToString(), "Cannot submit a bid for project that is not available for bids."));
+            if (storedProject.Status != Constants.PROJECT_STATUS_PENDING)
+                return Conflict(CreateErrorResponse(StatusCodes.Status409Conflict.ToString(), "Cannot submit a bid for project that is not Pending for bids."));
```

Comment:
"Cannot submit a bid for project that is not Pending for bids."
Add `that is not Pending or Completed`

Reply:
done !